app/BotMan/ConfigParser.php

```diff
@@ -37,10 +45,59 @@ public function configure(BotMan $bot)
      *
      * @param string $reply
      */
-    private function reply(string $reply)
+    private function addReply(string $reply) : Callable
     {
         return function (BotMan $bot) use ($reply) {
```

Comment:
How does `$bot` get injected into the `addReply()` scope?

Reply:
`addReply` returns a callable accepting the `$bot` argument.

```
$func = $parser->addReply('foo');

$func($bot);
```